var/spack/repos/builtin/packages/opium/package.py

```diff
@@ -37,11 +37,9 @@ class Opium(Package):
     depends_on('lapack')
 
     def install(self, spec, prefix):
+        libs = spec['lapack'].lapack_libs + spec['blas'].blas_libs
```

Comment:
i wonder if the unit test should behave like

```
libs = libs + libs
```
i.e. if you add two similar sets of libs (in certain order), the result should be the same set of the same size?  In other words, make `+` to operate like a union between sets.